mycore-mods/src/test/java/org/mycore/mods/merger/MCRCategoryMergeEventHandlerTest.java

```diff
@@ -0,0 +1,81 @@
+package org.mycore.mods.merger;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.jdom2.Document;
+import org.jdom2.Element;
+import org.jdom2.JDOMException;
+import org.jdom2.input.SAXBuilder;
+import org.junit.Test;
+import org.mycore.common.MCRConstants;
+import org.mycore.common.MCRJPATestCase;
+import org.mycore.common.MCRSessionMgr;
+import org.mycore.common.MCRTransactionHelper;
+import org.mycore.common.content.MCRJDOMContent;
+import org.mycore.datamodel.classifications2.MCRCategory;
+import org.mycore.datamodel.classifications2.MCRCategoryDAO;
+import org.mycore.datamodel.classifications2.MCRCategoryDAOFactory;
+import org.mycore.datamodel.classifications2.utils.MCRXMLTransformer;
+import org.mycore.datamodel.metadata.MCRObject;
+import org.mycore.mods.MCRMODSWrapper;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class MCRCategoryMergeEventHandlerTest extends MCRJPATestCase {
+
+    public static final String TEST_DIRECTORY = "MCRCategoryMergeEventHandlerTest/";
+
+    private static final Logger LOGGER = LogManager.getLogger();
+
+    public MCRCategoryDAO getDAO() {
+        return MCRCategoryDAOFactory.getInstance();
+    }
+
+    @Override
+    public void setUp() throws Exception {
+        super.setUp();
+    }
+
+    /**
+     * Tests if parent genres are succesfully removed from mods.
+     */
+    @Test
+    public void testHandleObjectCreatedMultipleGenres() throws IOException, JDOMException, URISyntaxException {
+
+        MCRSessionMgr.getCurrentSession();
+        MCRTransactionHelper.isTransactionActive();
+        ClassLoader classLoader = getClass().getClassLoader();
+        SAXBuilder saxBuilder = new SAXBuilder();
+
+        MCRCategory category = MCRXMLTransformer
+            .getCategory(saxBuilder.build(classLoader.getResourceAsStream(TEST_DIRECTORY + "genre.xml")));
```

Comment:
Should rather use `MCRResourceHelper#getResourceAsStream(String)` instead of `ClassLoader#getResourceAsStream(String)` (two occurances).